lib/dry/rails/boot/safe_params.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   start do
-    ApplicationController.include(Dry::Rails::Features::SafeParams)
+    ActionController::Base.include(Dry::Rails::Features::SafeParams)
```

Comment:
I *suspect* this will be problematic when there are multiple engines with safe params enabled because it will result in including this feature in `ActionController::Base` multiple times. I've no idea how to solve this 😅

Reply:
ouch, missed that, thanks for catching it  😄 
do you think we could work around it with `Module#included?`, like: 

```ruby
unless ActionController::Base.include?(Dry::Rails::Features::SafeParams)
  ActionController::Base.include(Dry::Rails::Features::SafeParams
end
```

Not the most elegant maybe, but it should work 😄

Reply:
@zlw yes, guarding using `included?` would do the trick for now, maybe later we'll find a nicer solution 🙂